website/docs/r/kubernetes_cluster.html.markdown

```diff
@@ -891,7 +891,7 @@ A `sysctl_config` block supports the following:
 
 A `web_app_routing` block supports the following:
 
-* `dns_zone_id` - (Required) Specifies the ID of the DNS Zone in which DNS entries are created for applications deployed to the cluster when Web App Routing is enabled. For Bring-Your-Own DNS zones this property should be set to an empty string `""`.
+* `dns_zone_ids` - (Required) Specifies the list of the DNS Zone IDs in which DNS entries are created for applications deployed to the cluster when Web App Routing is enabled. For Bring-Your-Own DNS zones this property should be set to an empty string `""`.
```

Comment:
> For Bring-Your-Own DNS zones this property should be set to an empty string `""`

Is that still valid?

Reply:
I double checked with service team, and it should be an empty list now. Actually, user should set it to empty string/list if **NOT** using the Bring-Your-Own DNS zones feature. I've fixed it in this PR.